### ISSUE TITLE
Discussion: Is there a plan for Coloured Icons for FluentUI Icons?

### ISSUE THREAD

**Issue by mdtauk** [microsoft/fluentui-system-icons#275]
I know, I know, obligatory "Apple is doing something so shouldn't Microsoft..." But there are some things to consider.

https://developer.apple.com/design/human-interface-guidelines/sf-symbols/overview/
![image](https://user-images.githubusercontent.com/7389110/121078648-20f5bd00-c7d1-11eb-8e88-096dd3bba4ea.png)

Apple's v3 of their SF Symbols icons/iconfont project has added more colour layers for their icons.

At present, Fluent UI System Icons are monocoloured.  But Office and Visual Studio currently do, or plan to, adopt icons with colours.  So is there a need to add some consideration to how the icons work with colour?

### Apps

![image](https://user-images.githubusercontent.com/7389110/121079502-1a1b7a00-c7d2-11eb-9f4e-3288b39768f1.png)

![image](https://user-images.githubusercontent.com/7389110/121079887-a5950b00-c7d2-11eb-9d95-2e0e3390cd15.png)

Apps used a layered and geometric Icon style with colour and gradient.  These are typically separate from the monolined icons, but in sympathy with their design. (both coming from a Fluent Design ethos)

But Office and Visual Studio has a middle ground, colour whilst still monoline.
![image](https://user-images.githubusercontent.com/7389110/121079352-eccecc00-c7d1-11eb-8d02-d95d2980b180.png)

![image](https://user-images.githubusercontent.com/7389110/121079126-abd6b780-c7d1-11eb-82f5-a6520a4e7afa.png)

**Comment by spencer-nelson** [comment]
TL;DR; We're looking at ways to provide multicolor icons.

**Comment by mdtauk** [comment]
Just to share what the "competition" are doing lol

Apple's SF Symbols will support Monochrome, Hierarchical, and Palette "render modes" for their icons.

![image](https://user-images.githubusercontent.com/7389110/121393886-b071ac00-c948-11eb-827c-137bd2e74504.png)

Monochrome and Hierarchical take a single foreground colour, and applies tints
![image](https://user-images.githubusercontent.com/7389110/121394028-d7c87900-c948-11eb-81fd-2666db419343.png)

Palette applies a colour to each "layer"
![image](https://user-images.githubusercontent.com/7389110/121394115-ed3da300-c948-11eb-8689-4eefde7f928b.png)

**Comment by YBAZAN** [comment]
Any news on this point ? Can we expect this feature one day ?
